mobile/src/core/enums.ts

```diff
@@ -10,3 +10,10 @@ export enum KeyChainKeys {
   passcodeEnteredTime = 'PASSCODE_ENTERED_TIME',
   settings = 'SETTINGS'
 }
+
+export enum AsyncParticleStates {
+  IDLE = 0,
+  LOADING = 1,
+  SUCCESS = 2,
+  FAILED = 3
```

Comment:
Curious about the use of the word "particle" - is this a redux convention?

Reply:
Don't think so